packages/istanbul-lib-source-maps/test/transformer.test.js

```diff
@@ -53,6 +41,35 @@ function createData() {
     };
 }
 
+function createDataBackslash() {
+    var coverageData = {
+        path: '\\path\\to\\file.js',
```

Comment:
For now we could use `Object.assign` here to save some of the repetition. As far as I can tell this is the only line that changes b/w these two fns so we could have a common `const` that is shared in a helper file or just at the top of this one.

Reply:
Yeah, I'll try to reduce the amount of duplicated test data!

Reply:
Done!

Reply:
Btw, I used a `var` since everything else is declared in a single `var` at the top, maybe the whole file should be updated to use `const`, `let` etc?

Reply:
I haven't had a chance to look at your updated PR yet but I'd say leave it on `var` for this PR.  Switching to `const` / `let` would be a nice follow-up.  It would be a 'mass change' that could be easily reviewed on it's own but would bulk this PR.

Reply:
@coreyfarrell I think it would be worth making the `const` `let` change as we go, instead of leaving it as tech debt to be cleaned up later. Gradual improvements as PRs are made make the most sense IMHO

Reply:
Ah yes using `const` / `let` for new or modified code does make sense.  And on second look this file is not so large so I guess updating this whole file wouldn't be a big deal.

Reply:
Ok, the test file is converted to `const` 👍